docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+| quote mark  | character  | entity reference | character reference |
+|---|---|---|---|
+| left double | &ldquo;  | `&ldquo;`  | `&#8220;`
+| right double | &rdquo;  | `&ldquo;`  | `&#8221;`
+| left single | &lsquo;  | `&ldquo;`  | `&#8216;`
+| right single | &rsquo;  | `&ldquo;`  | `&#8217;`
+
+Preview carefully when using these. Another thing that works in modern browsers `<q>` element markup, like `<q>this</q>`, which our Markdown interpreter renders through to the browser as <q>this</q>.
+
+### Signing work
+
+Our work products are in the public domain, and for the most part we rely on git and Github to track contributions, and consider all works to be created and produced by the OSCAL team.
+
+However, for authors of discrete articles, pieces of software or other research findings or work products, where names should be listed for any reason, this can be accommodated. This applies generally, to project developers and teams as well.
+
+### Simple English and plain language
+
+This is an important topic covered in the style guides listed below. See also [US Government resources on plain language](https://www.plainlanguage.gov/).
+
+Technical terminology alone -- both OSCAL terminology and the terminologies of its domains of application --  will sometimes make documentation difficult for some readers. But this is not a goal, and terminology aside, sentences should be kept short. When choosing between two ways to express the same thing, choose the simpler and commoner one.
+
+### Technical terminology
+
+When writing about OSCAL or topics related to OSCAL, technical terminology is indispensable and inescapable. Additionally, it is inevitable and foreseeable that some terms (examples that come readily to mind include <q>control</q>, <q>catalog</q>, <q>profile</q>, <q>constraint</q>, and <q>component</q>) will be overloaded in use, that is, they will be used in both technical and non-technical senses, sometimes within the same conversation. Moreover their various technical senses may sometimes be broadly *contradictory* (what is meant by <q>profile</q> might be specifically different from one technical context to another) without ever creating *ambiguity*, if their contexts of use are kept clear. Use technical terminology to prevent ambiguity, and where ambiguity arises, clarify context in order to clarify usage.
+
+For example, whenever writing about <q>validation</q>, be specific as to what kind of validation is meant, since the word has both generic and specific senses.
+
+Similarly, <q>control</q> has both an OSCAL sense, and an RMF sense (Risk Management Framework as defined by NIST SP800-37); a document that uses the term specifically in one of these senses, the other, or both (when OSCAL is being used to model RMF), might indicate this.
```

Comment:
The point made in this paragraph is excellent, and I believe Dave suggested we will apply what we quickly described as "use a specific formal noun, at least two words rule: OSCAL Control, RMF Control, but not control; XML Schema validation, OSCAL model validation, but not just validation" Can we reference that heuristic in the project? I happen to like that as it is a simple measure for these kinds of things.

Reply:
I like it well enough but I think it is best as a "folk heuristic". I don't think people should be penalized or ostracized for saying "control" after they've said "OSCAL Control" six or eight times in a conversation....